File: TouchInsightAdmin/ViewController.m
//
//  ViewController.m
//  TouchInsightAdmin
//
//  Created by Touch on 11/27/2558 BE.
//  Copyright © 2558 weerapons suwanchatree. All rights reserved.
//

#import "ViewController.h"
#import <SMPageControl/SMPageControl.h>

@interface ViewController ()
{
    UIView *rootView;
  EAIntroView *_intro;
}
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (void)showIntroWithCrossDissolve {

}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

- (IBAction)click:(id)sender {
    EAIntroPage *page1 = [EAIntroPage page];
    page1.title = @"Hello world";
    page1.desc = @"qwertyuiop";
    page1.bgImage = [UIImage imageNamed:@"bg1"];
    page1.titleIconView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"title1"]];
    
    EAIntroPage *page2 = [EAIntroPage page];
    page2.title = @"This is page 2";
    page2.desc =  @"qwertyuiop";
    page2.bgImage = [UIImage imageNamed:@"bg2"];
    page2.titleIconView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"title2"]];
    
    EAIntroPage *page3 = [EAIntroPage page];
    page3.title = @"This is page 3";
    page3.desc =  @"qwertyuiop";
    page3.bgImage = [UIImage imageNamed:@"bg3"];
    page3.titleIconView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"title3"]];
    
    EAIntroPage *page4 = [EAIntroPage page];
    page4.title = @"This is page 4";
    page4.desc =  @"qwertyuiop";
    page4.bgImage = [UIImage imageNamed:@"bg4"];
    page4.titleIconView = [[UIImageView alloc] initWithImage:[UIImage imageNamed:@"title4"]];
    
    EAIntroView *intro = [[EAIntroView alloc] initWithFrame:rootView.bounds andPages:@[page1,page2,page3,page4]];
    
    [intro setDelegate:self];
    
    [intro showInView:rootView animateDuration:0.3];
    
   // [self showIntroWithCrossDissolve];

}
@end

File: TouchInsightAdmin/ViewController.swift
//
//  ViewController.swift
//  TouchInsightAdmin
//
//  Created by weerapons suwanchatree on 11/25/2558 BE.
//  Copyright © 2558 weerapons suwanchatree. All rights reserved.
//

import UIKit
import Alamofire
//import ObjectMapper
//import AlamofireObjectMapper


class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    @IBAction func gogogo(sender: AnyObject) {
        
        let headers = [
            "X-Parse-REST-API-Key":"sqfSdSWsRtZFCvMtYuPo3Rjum72wceAW4Jyan2Zp",
            "X-Parse-Application-Id":"lxPvvAlKnkn7xS4mUAuPyxEBhAMTh4JYYKOCYV6P",
            "Content-Type":"application/json",
            ]
        
        // Add URL parameters
        let urlParams = [
            "":"{\"username\":{\"$in\":[\"tak\"]},\"password\":{\"$in\":[\"1234\"]}}",
            "where":"{\"username\":{\"$in\":[\"tak\"]}}",
            ]
        
        // Fetch Request
        Alamofire.request(.GET, "https://parseapi.back4app.com/classes/member", headers: headers, parameters: urlParams)
            .validate(statusCode: 200..<300)
            .responseJSON { response in
                if (response.result.error == nil) {
                    debugPrint("HTTP Response Body: \(response.data)")
                }
                else {
                    debugPrint("HTTP Request failed: \(response.result.error)")
                }
        }
        
    }

    
    

}


